readme.txt

```diff
@@ -312,9 +312,10 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 
 * New - Introduced the `tribe_events_query_force_local_tz` filter to allow for forcing non-UTC event start and end times in Tribe__Events__Query [92948]
 * Fix - Allow The Events Calendar REST API to be disabled using the `tribe_events_rest_api_enabled` filter [97209]
-* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659] 
+* Tweak - Added some more context to the labeling of the "Number of events per page" option (thanks to Todd H. for highlighting this label) [73659]
 * Tweak - Improve performance on Event Admin List Count by removing JOIN and use cached results [63567]
 * Tweak - Made the "/page/" component of some views' URL string translatable [40976]
+* Tweak - Allow translations on the "events" and "event" slugs.
```

Comment:
Hate to nitpick @mitogh, but wanted to call out our specific format for these changelog entries:

- First, don't end with a period
- Next, ensure you have the ticket number in square brackets at the end
- Next, this isn't a big deal, but I would say to move this up to the top of the list and change it from `Tweak` to `New` — this seems like a new feature to me, and one worth highlighting as such
- Finally, in your code you highlight that the slugs are only translatable with WPML or other multilingual plugins. I think that's worth clarifying here. So instead of this:
```
Tweak - Allow translations on the "events" and "event" slugs.
```
Maybe something like this:
```
New - Made the "events" and "event" slugs translatable by WPML and other multilingual plugins [95026]

Reply:
Thanks a lot guys your feedback is really appreciated 🔝  ⭐️